### PR TITLE
Reconstruction of components , style désktɒp, mobile, tablet

### PR DESCRIPTION
**Pull request by ira089**

### CODE REVIEW

File: src/components/common/Login/Login.tsx [hunk 1]
```diff
@@ -1,18 +1,14 @@
...
 import { Button } from "@/components/ui";
 import { AppRoutes } from "@/constants";
 import { Link } from "react-router-dom";
-import { LoginForm } from "@/components/common/LoginForm";
-import styles from "../SignInPage/SignInPage.module.scss";
+import { LoginForm } from "@/components/common/Login";
+import styles from "./Login.module.scss";
 
-export const LoginPage = () => {
+export const Login = () => {
```
Comment (Eugene-Superman) [inline src/components/common/Login/Login.tsx:9]: there are 2 exports in this component

File: src/components/common/Register/Register.tsx [hunk 2]
```diff
@@ -1,15 +1,11 @@
-import logo from "@/assets/images/logo.png";
 import { AppRoutes } from "@/constants";
 import { Link } from "react-router-dom";
-import { RegisterForm } from "@/components/common/RegisterForm";
-import styles from "../SignInPage/SignInPage.module.scss";
+import { RegisterForm } from "@/components/common/Register";
+import styles from "./Register.module.scss";
 
-export const RegistrationPage = () => {
+export const Register = () => {
```
Comment (Eugene-Superman) [inline src/components/common/Register/Register.tsx:6]: 2 exports

File: src/components/common/refresh/RefreshForma.tsx [hunk 3]
```diff
@@ -3,16 +3,15 @@ import { Button, InputFormik } from "@/components/ui";
...
-
-interface Values {
+export interface IRefreshData {
   email: string;
   password: string;
   confirmpassword: string;
 }
 
-export const RefreshForma = () => {
+const RefreshForma = () => {
```
Comment (Eugene-Superman) [inline src/components/common/refresh/RefreshForma.tsx:14]: typo: correct is RefreshForm, not Forma

File: src/pages/ForgotPasswordPage.tsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { ForgotPassword } from "@/components/common/ForgotPassword";
```
Comment (Eugene-Superman) [inline src/pages/ForgotPasswordPage.tsx:1]: Is this component necessary? It doesn't have any functionality/logic. You can use ForgotPassword without ForgotPasswordPage

The same question for: src/pages/LoginPage.tsx, src/pages/RefreshPage.tsx, src/pages/RegistrationPage.tsx, src/pages/SecurePage.tsx,

Comment (ira089) [reply src/pages/ForgotPasswordPage.tsx:1]: такому подходу нас учили на курсах - в page нет логики - вся логика в компонентах

File: src/components/common/Login/Login.tsx (hunk 1 above)
Comment (ira089) [reply src/components/common/Login/Login.tsx:9]: исправила

File: src/components/common/Register/Register.tsx (hunk 2 above)
Comment (ira089) [reply src/components/common/Register/Register.tsx:6]: исправила

File: src/components/common/refresh/RefreshForma.tsx (hunk 3 above)
Comment (ira089) [reply src/components/common/refresh/RefreshForma.tsx:14]: исправила

**Review by l-dmitry94 — APPROVED** [review]

**Pull request merged**